damnit/backend/db.py

```diff
@@ -29,7 +30,8 @@ def __init__(self, path=DB_NAME):
         log.debug("Opening database at %s", path)
         self.conn = sqlite3.connect(path, timeout=30)
         # Ensure the database is writable by everyone
-        os.chmod(path, 0o666)
+        if Path(path).owner() == getuser():
```

Comment:
```suggestion
        if Path(path).stat().st_uid == os.getuid():
```

This should be equivalent, but it avoids some possible failure modes from translating the numeric user ID into a readable username. I'm thinking of possibilities like the user who created it no longer exists on the system, or it's created from a user inside a container. It may not be possible at all, but it's easier to avoid the lookup than to figure out if it could really come up.

Reply:
Ah, good point; fixed in a938798.